server/src/main/java/io/spine/server/aggregate/AggregateStorage.java

```diff
@@ -104,7 +104,32 @@ protected AggregateStorage(boolean multitenant) {
         }
         builder.addAllEvent(history);
 
-        return Optional.of(builder.build());
+        final AggregateStateRecord result = builder.build();
+        checkAggregateStateRecord(result);
+        return Optional.of(result);
+    }
+
+    /**
+     * Ensures that the {@link AggregateStateRecord} is valid.
+     *
+     * <p>{@link AggregateStateRecord} is considered valid when one of the following is true:
+     * <ul>
```

Comment:
As discussed vocally, my comment is irrelevant. Since previously there were no indentation, I confused `<ul>` with another `<li>` by mistake.

Thanks for standing this up.